### PR TITLE
Inital code

### PR DESCRIPTION
**Pull request by Text-Input**
A base data collector to build off of.

### CODE REVIEW

**Comment by buzuloiu** [comment]
You should consider generating a `requirements.txt` with something like pipreqs: 
https://github.com/bndr/pipreqs#example

File: code/dataq_DI1100_Test.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
+"""
```
Comment (buzuloiu) [inline code/dataq_DI1100_Test.py:1]: Is this file from [here](https://github.com/dataq-instruments/Python/blob/master/binary_comm/DI-1100/DI_1100.py)?

File: code/csv_test.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+for i in range(0,40000):
+    f.write(str(i) + ";\n") #note, to get this speed, we can't
+                            #print every piece of data we get.
+
+
+end = time.time()
+
+print("it took", (end-start)*1000, "ms to write all the data.")
+
+f.write(str((start-end)*1000))
```
Comment (buzuloiu) [inline code/csv_test.py]:
```suggestion
f.write(str((end-start)*1000))
```

File: code/csv_test.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#testing writing to CSV files. making sure that we can write fast enough.
+
+import time
+
+f = open("test.txt", "w+")#write to file, if not there, create it.
```
Comment (buzuloiu) [inline code/csv_test.py:5]:
```suggestion
f = open("%s-test.txt" % time.time(), "w+")#write to file, if not there, create it.
```
I don't have a strong opinion about this, but you could keep all of your test files separate by creating them with a timestamp.

**Review by buzuloiu — APPROVED** [review]
Looks good! Just a couple suggestions, but other than that it's good to go.

**Comment by Text-Input** [comment]
As a note, the 'keyboard' package seems to be causing some issues, so it may be removed in the future.

File: code/dataq_DI1100_Test.py (hunk 1 above)
Comment (Text-Input) [reply code/dataq_DI1100_Test.py:1]: Yes it is. Is there a specified way to reference it's origin? or should I just put the link in the file at the start?

Comment (buzuloiu) [reply code/dataq_DI1100_Test.py:1]: No, I was just curious! If anything either add a link in the README or create a `documentation` page in the wiki for this repo and post the link to the repo there so we can keep track of the resources for what we use.

Comment (Text-Input) [reply code/dataq_DI1100_Test.py:1]: That sounds like a good idea. I'll add it as a reference to the README then.

File: code/csv_test.py (hunk 3 above)
Comment (Text-Input) [reply code/csv_test.py:5]: That sounds like a good idea to do. The 'test.txt' was simply temporary. I was planning on implementing something to differentiate them, and this seems like as good an idea as any. I think that I'll implement that in the main files. The file you linked to was a test file for testing if python could write to a file fast enough. If you don't think that's something we need to keep around, the file doesn't need to be kept.

Comment (buzuloiu) [reply code/csv_test.py:5]: Oh ok that makes sense, yeah it's fine to keep it. Maybe when you start writing the main code move everything that you're just using for testing/demos into a separate folder in the repo just to keep it separate from the real code.

Comment (Text-Input) [reply code/csv_test.py:5]: Sure thing!
I'll do that once the main code starts getting written.

**Pull request merged**